server/android/src/main/AndroidManifest.xml

```diff
@@ -26,7 +28,11 @@
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
+                <action android:name="android.hardware.usb.action.USB_DEVICE_ATTACHED" />
             </intent-filter>
+
+            <meta-data android:name="android.hardware.usb.action.USB_DEVICE_ATTACHED"
+                android:resource="@xml/device_filter" />
```

Comment:
One note about this, it seems that the "Open SlimeVR" popup shows up regardless of whether the app is already running, kinda annoying ngl